quiztest/src/main/java/quiztest/main/inventory/InventoryService.java

```diff
@@ -0,0 +1,220 @@
+package quiztest.main.inventory;
```

Comment:
A general comment for the class. I recommend keeping all scheduled tasks near one another towards the bottom, but before afterPropertiesSet so the core logic other classes can use is grouped together.

Additionally, I would probably recommend migrating the scheduled methods out of this class entirely and putting them into their own class and I would make that class only load in your respective development environments and **not** on the prod profile.

Reply:
DealershipSimulator class created and scheduled tasks migrated to it.